From: Heiko Stuebner <heiko@sntech.de>
Subject: [PATCH] Force dot threads

Force dot to use only one thread, to prevent segfaults on armel.

Signed-off-by: Heiko Stuebner <heiko@sntech.de>

---
 doc/Doxyfile    |    2 +-
 doc/Doxyfile.in |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/Doxyfile b/doc/Doxyfile
index 00fc814..9de6ce4 100644
--- a/doc/Doxyfile
+++ b/doc/Doxyfile
@@ -1483,7 +1483,7 @@ HAVE_DOT               = YES
 # explicitly to a value larger than 0 to get control over the balance
 # between CPU load and processing speed.
 
-DOT_NUM_THREADS        = 0
+DOT_NUM_THREADS        = 1
 
 # By default doxygen will write a font called FreeSans.ttf to the output
 # directory and reference it in all dot files that doxygen generates. This
diff --git a/doc/Doxyfile.in b/doc/Doxyfile.in
index c183abd..ce55215 100644
--- a/doc/Doxyfile.in
+++ b/doc/Doxyfile.in
@@ -1483,7 +1483,7 @@ HAVE_DOT               = YES
 # explicitly to a value larger than 0 to get control over the balance
 # between CPU load and processing speed.
 
-DOT_NUM_THREADS        = 0
+DOT_NUM_THREADS        = 1
 
 # By default doxygen will write a font called FreeSans.ttf to the output
 # directory and reference it in all dot files that doxygen generates. This
-- 
tg: (782035e..) fixes/dot-threads (depends on: upstream-dist)
